src/dmd/dtoh.d

```diff
@@ -332,6 +396,8 @@ public:
         Other
     }
 
+    alias AST = ASTCodegen;
```

Comment:
Why this ? Seems unrelated.

Reply:
oh that. allows my editor to jup to the definitions of the AST structs.

Reply:
I can back this change out of you take offence at it

Reply:
It's a functional change completely unrelated to indentation, so yes, please remove